docs/cheatsheet.rst

```diff
@@ -46,6 +46,12 @@ Members of ``address``
 - ``<address>.balance`` (``uint256``): balance of the :ref:`address` in Wei
 - ``<address>.code`` (``bytes memory``): code at the :ref:`address` (can be empty)
 - ``<address>.codehash`` (``bytes32``): the codehash of the :ref:`address`
+- ``<address>.call(bytes memory) returns (bool, bytes memory)``: issue low-level ``CALL`` with the given payload implemented by :ref:`address`,
```

Comment:
This entry should be before `<address>.code`.

Reply:
Do you think it's good to arrange things in alphabetical order like this? Can't methods and properties be put together? By the way, the other parts are not completely in alphabetical order, why bother with this on a document that is overall very poor?

Reply:
Yeah, I didn't think about maintaining the methods and properties grouped when I suggested that. You have a good point. Let's keep it as is.



> why bother with this on a document that is overall very poor?

Hopefully we can improve that, bit by bit, with the help of the community :)